API.md

```diff
@@ -205,80 +205,105 @@ The three drawing modes work identically. They do not take an options argument.
 
 ## Events
 
-Draw fires off a number of events. All of these events are name spaced `draw.` and are emitted via map object.
+Draw fires off a number of events. All of these events are namespaced with `draw.` and are emitted from the map object.
 
-### draw.modechange
+They are all triggered as the result of user interaction. Programmatically manipulating Draw (with the function documented above) will not fire these events.
 
-This event is fired just after the current mode is stopped and just before the next mode is started. A render will not happen until after all event handlers have been triggered. This means you can force a mode redirect by calling `changeMode` inside of a `draw.modechange` handler.
+### `draw.create`
 
-This is not fired when the first mode is started.
+Fired when a feature is created as the result of a user interaction. The following interactions will trigger this event:
 
-Here is an example payload.
+- Finish drawing a feature. Simply clicking will create a Point. A LineString or Polygon is only created when the user has finished drawing it — i.e. double-clicked the last vertex or hit Enter — and the drawn feature is valid.
+
+The event data is an object with the following shape:
 
 ```js
 {
-  mode: `direct_select`
-  opts: '123123123'
+  // Array of GeoJSON objects representing the features that were created
+  features: [{..}, {..}, ..]
 }
 ```
 
-### draw.deleted
+### `draw.delete`
 
-This is fired every time a feature is deleted. The payload is a list GeoJSON features just before they were deleted.
+Fired when one or more features are deleted as the result of a user interaction. The following interactions will trigger this event:
 
-### draw.changed
+- Click the Trash button when one or more features are selected in `simple_select` mode.
+- Hit the Backspace or Delete keys when one or features are selected in `simple_select` mode.
 
-This is fired when feature coordinates are changed. It is fired just after the changes have been sent to be rendered. The payload is the GeoJSON of all changed features. Deleted features will not show up here.
+The event data is an object with the following shape:
 
 ```js
 {
-  "features": [{ ... }]
+  // Array of GeoJSON objects representing the features that were deleted
+  features: [{..}, {..}, ..]
 }
 ```
 
-### draw.modified
+### `draw.update`
+
+Fired when one or more features are updated as the result of a user interaction. The following interactions will trigger this event, with different `type` properties in the event data:
+
+- Finish moving one or more selected features in `simple_select` mode. The event will only fire when the movement is finished — i.e. when the user releases the mouse button or hits Enter. Type: `'move'`.
```

Comment:
Should we start these with the type (ie: move) and then have the description of when it will happen?